### ISSUE TITLE
codes

### ISSUE THREAD

**Issue by ADITYA-THEE-CODER** [ADITYA-THEE-CODER/ML-#80]
scaler = StandardScaler()
scaled_data = scaler.fit_transform(data)

model2 = KNeighborsClassifier(n_neighbors=3)
model2.fit(scaled_data, labels)

# IMPORTANT: scale input too
new_point = scaler.transform([[28, 60000]])

pred2 = model2.predict(new_point)
print("With Scaling:", pred2)
